src/core/data/server.c

```diff
@@ -96,22 +97,47 @@ _server_pipe_write(void)
 }
 #endif
 
+static inline void
+_server_notify_worker(void)
+{
+#ifdef USE_EVENT_FD
+    _server_event_fd_write();
+#else
+    _server_pipe_write();
+#endif
+}
+
 /* pipe_read recycles returned streams from worker thread */
 static inline void
-_server_pipe_read(void)
+_server_read_notification(void)
```

Comment:
For naming consistency, shall we call the two notification functions `_server_send_notification` and `_server_read_notification`?

Reply:
I meant `write` and `read`